openedx_events/bridge/tests/test_avro_attrs_bridge.py

```diff
@@ -9,7 +9,14 @@
 
 from openedx_events.bridge.avro_attrs_bridge import AvroAttrsBridge
 from openedx_events.bridge.avro_attrs_bridge_extensions import CourseKeyAvroAttrsBridgeExtension
-from openedx_events.learning.data import CourseData, CourseEnrollmentData, UserData, UserPersonalData
+from openedx_events.learning.data import (
+    CourseData,
+    CourseEnrollmentData,
+    UserData,
+    UserPersonalData,
+)
+
+from openedx_events.tooling import OpenEdxPublicSignal
```

Comment:
This needs tests around making sure that this constructs a proper schema from a signal definition.